sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/ServiceAPITest.groovy

```diff
@@ -291,6 +292,7 @@ class ServiceAPITest extends APISpec {
         thrown(StorageException)
     }
 
+    @Ignore
```

Comment:
Why are these tests being ignored now?

Reply:
As below.

Reply:
Can you please provide more context on this - I don't know why they are being ignored either.

Reply:
I have a PR for addressing the issue. 
We can uncomment the ignore once the PR gets in.
https://github.com/Azure/azure-sdk-for-java/pull/5176

Reply:
User delegate key is asking service principle ids.
We use our personal account previously to get the tests passed. However, the live tests are using the team account which use other service principle id and key.
The PR above is to update the key and add missing creds we need to use.
Once the PR got merged in, live tests should pass and I will uncomment here.